src/actions/types.js

```diff
@@ -7,7 +7,9 @@ export const APP_SHOW_ONBOARDING = '[app] SHOW_ONBOARDING'
 export const APP_GDPR_FAILED = '[app] GDPR_FAILED'
 export const APP_USER_HAS_INACTIVE_TOKEN = '[app] USER_HAS_INACTIVE_TOKEN'
 export const APP_USER_NIGHT_MODE_SAVE = '[app] USER_NIGHT_MODE_SAVE'
+export const APP_USER_NIGHT_MODE_SAVE_ERROR = '[app] USER_NIGHT_MODE_ERROR'
 export const APP_USER_BETA_MODE_SAVE = '[app] USER_BETA_MODE_SAVE'
+export const APP_USER_BETA_MODE_SAVE_ERROR = '[app] USER_BETA_MODE_SAVE_ERROR'
```

Comment:
To preserve consistency across the codebase, it's better to end `error` name with a [`FAILED` postfix](https://github.com/santiment/app/pull/457/files#diff-fbed0602d73913936d71c47e897c48c5R69)